bokeh/core/properties.py

```diff
@@ -1255,6 +1263,23 @@ def from_json(self, json, models=None):
         else:
             raise DeserializationError("%s expected a dict or None, got %s" % (self, json))
 
+class ColumnData(Dict):
+    """Property holding column data in form of a dict. Also applies
+    encoding and decoding to the data.
+    """
```

Comment:
Alternatively, put the virtualization of from_json all here in this `ColumnData` class ... use `Seq(Any)` but in the from_json for this `ColumnData` class, do not call `from_json` on the values immediately; instead, first check whether they are base64-encoded arrays and if so handle them directly, and if not call their `from_json`. So all base64-special-casing would be here in `ColumnData`.